aspnet-core/src/HRMv2.Core/Manager/BaseManager.cs

```diff
@@ -116,5 +119,33 @@ public string GetSurNameByFullName(string fullName)
         {
             return fullName.Substring(0, fullName.LastIndexOf(" "));
         }
+
+        public void CheckViewOrViewMyBranchEmployee(string permission)
+        {
+            if (IsGranted(permission)) throw new AbpAuthorizationException($"Required permissions are not granted. [{permission}]");
+        }
+        public void CheckEmployeeInCurrentBranch(long employeeId)
+        {
+            bool isViewAll = IsGranted(PermissionNames.Employee_View);
+            if (!isViewAll)
+            {
+                var currentUserBranch = GetBranchByCurrentUser();
```

Comment:
cần check thêm quyền view employee of my  branch nữa chứ

Reply:
dạ, ở bên ngoài em gán cho API là phải có 1 trong 2 quyền mới gọi đến API này rồi ạ, ví dụ như trong ảnh dưới, thì nó sẽ check 1 trong 2 quyền ở attribute trên ok rồi nó check tiếp đến quyền ở attribute dưới ạ
![image](https://github.com/ncc-erp/ncc-erp-hrm-v2/assets/144658145/4e2994ca-acfd-46f8-a00b-2dd80d8f11f7)